standard/docs/en/support/governance.md

```diff
@@ -126,49 +128,49 @@ If there are at least two parties interested in using an extension, and followin
 
 ### Development & docs
 
 The technical team, working with community members, will work on a development branch to prepare updates to the schema, documentation and codelists, according to the prioritized list. 
 
 This stage is likely to involve broad community engagement and discussion of specific decisions through GitHub issues. 
 
-At the point where all open issues are suitably addressed, the development branch can be submitted for peer review.
+At the point where all open issues are suitably addressed, the development branch may be submitted for peer review.
 
 ### Peer-review
 
 The updated schema, documentation along with a changelog and narrative description of the changes will be released for peer-review.
 
 A group of invited reviewers, representing different stakeholders, and including data publishers and users, will be asked to complete a full review of the changes, and to submit:
 
-* A judgment on whether the overall upgrade, and/or specific changes should be **accepted**, **accepted with minor changes**, **substantially revised**, or **rejected**.
+* A judgment on whether the overall upgrade, and/or specific changes ought to be **accepted**, **accepted with minor changes**, **substantially revised**, or **rejected**.
 
 * Comments on each request for revisions or rejection.
 
-All reviews, with reviewer names, will be published. Community members may also submit their own reviews of the whole revision, or specific elements. The minimum period for peer-review is one month. 
+All reviews, with reviewer names, will be published. Community members can also submit their own reviews of the whole revision, or specific elements. The minimum period for peer-review is one month. 
 
 ### Revisions
 
-The technical team, with reference to the working group as appropriate, should evaluate reviews, and decide whether the whole upgrade, or specific features thereof, need to be revised, rejected or postponed to future processes.
+The technical team, with reference to the working group as appropriate, evaluates reviews, and decides whether the whole upgrade, or specific features thereof, need to be revised, rejected or postponed to future processes.
 
-If only minor changes are suggested, then the revised standard can be submitted back to reviewers for a brief review period of at least two weeks.
+If only minor changes are suggested, then the revised standard may be submitted back to reviewers for a brief review period of at least two weeks.
 
-If major changes are required, then a longer follow up review process of at least one month should be allowed for.
+If major changes are needed, then a longer follow up review process of at least one month should be allowed for.
```

Comment:
> If major changes are needed, then a longer follow up review process of at least one month **ought to** allowed for.

Reply:
"should" is retained to match above change.